docs/src/tutorial/minimal_working_compute.jl

```diff
@@ -0,0 +1,266 @@
+
+# At this point, we need to choose a queue family index to use. For this
+# example, have a look at `vulkaninfo` command and pick the good queue manually
+# from the list of `VkQueueFamilyProperties` -- you want one that has
+# `QUEUE_COMPUTE` in the flags. In a production environment, you would use
+# [`get_physical_device_queue_family_properties`](@ref) to find a good index.
+qfam_idx = 0
+
+# Create a device object and make a queue for our purposes.
+device = Device(physical_device, [DeviceQueueCreateInfo(qfam_idx, [1.0])], [], [])
+
+# ## Allocating the memory
+#
+# Same way, you need to find a good memory type. Again, you can find a good one
+# using `vulkaninfo` or with [`get_physical_device_memory_properties`](@ref).
+# For compute, you want something that is both at the device (contains
+# `MEMORY_PROPERTY_DEVICE_LOCAL_BIT`) and visible from the host
+# (`..._HOST_VISIBLE_BIT`).
+memorytype_idx = 0
+
+# Let's create some data. We will work with 100 flimsy floats.
+data_items = 100
+mem_size = sizeof(Float32) * data_items
+
+# Allocate the memory of the correct type
+mem = unwrap(allocate_memory(device, mem_size, memorytype_idx))
+
+# Make a buffer that will be used to access the memory, and bind it to the
+# memory. (Memory allocations may be quite demanding, it is therefore often
+# better to allocate a single big chunk of memory, and create multiple buffers
+# that view it as smaller arrays.)
+buffer = unwrap(create_buffer(device, mem_size, BUFFER_USAGE_STORAGE_BUFFER_BIT, SHARING_MODE_EXCLUSIVE, [qfam_idx]))
+
+bind_buffer_memory(device, buffer, mem, 0)
+
+# ## Uploading the data to the device
+#
+# First, map the memory and get a pointer to it.
+memptr = map_memory(device, mem, 0, mem_size)
+
+# Here we force Julia to look at the mapped data as at the vector of
+# `Float32`s, so that we can access it easily.
+data = unsafe_wrap(Vector{Float32}, convert(Ptr{Float32}, unwrap(memptr)), data_items, own = false)
+
+# For now, let's just zero out all the data, and *flush* the changes to make
+# sure the device can see the updated data. This is the simplest way to move
+# array data to the device.
+data .= 0
+flush_mapped_memory_ranges(device, [MappedMemoryRange(mem, 0, mem_size)])
+
+# Eventually, you may need to allocate memory types that are not visible from
+# host, because these provide better capacity and speed on the discrete GPUs.
+# At that point, you may need to use the transfer queue and memory transfer
+# commands to get the data from host-visible to GPU-local memory, using e.g.
+# [`cmd_copy_buffer`](@ref).
+
+# ## Compiling the shader
+#
+# Now we need to make a shader program. We will use `glslangValidator` packaged
+# in a JLL to compile a GLSL program from a string into a spir-v bytecode,
+# which is later passed to the GPU drivers.
+
+shader_code = """
+#version 430
+
+layout (local_size_x_id = 0) in;
+layout (local_size_x = 1, local_size_y = 1, local_size_z = 1) in;
```

Comment:
```suggestion
layout(local_size_x_id = 0) in;
layout(local_size_x = 1, local_size_y = 1, local_size_z = 1) in;
```
For consistency with other layout declarations. You can go for leaving a space instead of none in other declarations instead if you prefer.